Client/Application/AppDelegate.swift

```diff
@@ -335,6 +335,13 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         }
     }
 
+    func application(application: UIApplication, continueUserActivity userActivity: NSUserActivity, restorationHandler: ([AnyObject]?) -> Void) -> Bool {
+        if let url = userActivity.webpageURL {
```

Comment:
According to the API, we return true if we handle the URL and false otherwise. Might make sense to change this to return false outside the if statement but return true inside.